src/hooks.ts

```diff
@@ -7,8 +7,11 @@ export const handle: Handle = async ({ request, resolve }) => {
 
 	if (!cookies.session_id || cookies.session_id === 'deleted') request.locals.authenticated = false;
 	else {
+		const accessToken = await sessionDB.hget(cookies.session_id, 'accessToken');
```

Comment:
Is het `hget`? Weet niet of dat ding is van package oid. Je doet het overal, dus denk het wel

Reply:
Yes het is `hget` omdat ik een session als object opsla in redis (daar is een object een `hash` vandaar `'h'get`